### ISSUE TITLE
`oc start-build` complains about build that doesn't matter

### ISSUE THREAD

**Issue by stevekuznetsov** [openshift/origin#5242]
Following these steps:

```sh
$ oc create -f buildconfig.yaml
buildconfig "mybuild" created
$ oc start-build mybuild
mybuild-1
$ oc delete buildconfig
buildconfig "mybuild" deleted
$ oc create -f buildconfig.yaml
buildconfig "mybuild" created
$ oc start-build mybuild
Error from server: build "mybuild-1" already exists
$ oc delete build mybuild-1
build "mybuild-1" deleted
$ oc start-build mybuild
mybuild-2
```

Aside from the bad output test for `start-build` (bare object name without the other text), if the command is about to make `mybuild-2` I would not expect it to fail when `mybuild-1` already exists.

@csrwng

**Comment by csrwng** [comment]
@stevekuznetsov we are currently working on a card that should help with this. @mnagy owns it. When you delete the buildconfig, all builds associated with it should be deleted.

**Comment by mnagy** [comment]
Just an fyi, PR is here: #5208

**Comment by mnagy** [comment]
Resolved in #6185.
